Tests/FluentAssertions.Specs/Types/TypeAssertionSpecs.cs

```diff
@@ -1540,6 +1543,390 @@ public void When_injecting_a_null_predicate_into_TypeSelector_NotBeDecoratedWith
 
         #endregion
 
+        #region BeInNamespace
+
+        [Fact]
+        public void When_asserting_a_selection_of_types_in_a_namespace_is_in_that_namespace_it_succeeds()
```

Comment:
I would rewrite those tests like:

`When_a_type_is_in_the_expected_namespace_it_should_not_throw`

Same with the others.

Reply:
Fixed.